apps/mnmetrotransit/mn_metro_transit.star

```diff
@@ -0,0 +1,247 @@
+"""
+Applet: MN Metro Transit
+Summary: Train, BRT, ABRT, and Bus Departure Times
+Description: Shows Transit Departure Times from Selected Stop.
+Author: Alex Miller & Jonathan Wescott
+"""
+
+load("cache.star", "cache")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
+#Assign Default Stop Code
+DEFAULT_STOP_CODE = "15264"
+
+def main(config):
+    #Establish API URL
+    stop_code = config.get("stop_code", DEFAULT_STOP_CODE)
+    url = "https://svc.metrotransit.org/NexTripv2/" + stop_code + "?format=json"
+    MTT = http.get(url).json()
+
+    #Cache Data
+    MTT_cached = cache.get("MTT_rate")
```

Comment:
I think this `cache.get()` is meant to happen before the `http.get()`? Otherwise the http call will happen on each run, regardless of what's in cache.

Reply:
I've resolved the caching bug